modules/ROOT/pages/user-application-overview.adoc

```diff
@@ -1,60 +1,144 @@
 Alternatively, you can deploy an application by importing all its components in the following order:
 
 * Organization then profiles
 * API extensions then pages, layout, themes
 * Business Data Model then processes
 * Application descriptor
 
+Once deployed, log into Bonita and navigate to the application through the Application Directory.
+
 == Create your own User application from ours
 
-. Have all your custom pages ready, either imported in Bonita UI Designer or Bonita Studio.
-. Open our UI Designer pages to customize them.
-. In Bonita Studio, go to the "Project explorer" on the left and double-click on the "bonita-user-application.xml" application descriptor.
-. In the editor, rename the pages, change the order in the menu, remove the pages you do not need, add yours in the menu, or as orphan pages accessed through navigation from another page.
-+
-[WARNING]
+Creating your user application based on Bonita User Application means that you can change some of its existing pages, its navigation, its logo, its layout and/or its graphical theme and make those changes visible in the application descriptor. +
+
+=== Import it in Bonita Studio
+
+From Bonita Studio *Welcome page* _Resources_ tile, click on "Import Bonita User Application".
+
+image::images/UI2021.1/studio-get-application.png[Get application from Studio]
+
+You can also get it from our https://github.com/bonitasoft/bonita-admin-application/[Admin Application GitHub repository]. +
+There are two ways to get the .bos file. You can either get it from the _Releases_ on the right or by cloning the repository of your choice and building it using maven. +
+Building the repository will create a .bos file in the `target` folder. You can import this one into the development environment. +
+Using this method is also useful if you want to get an older version of an application or if you need a weekly version with a potential bugfix. +
+After validation, the application is imported or cloned; you can view its content in the project explorer, deploy it, and open it. +
+
+=== Change the existing pages
+
+. In the studio's Project Explorer, select the page to customize.
+ * All pages previously created using Google Web Toolkit framework have been recreated with Bonita UI Designer. They can be customized in the UI Designer. They come with the User Application you have download from Bonita Studio. You will see them in the Project Explorer.
+ * All pages previously created in other, more recent technologies, have been wrapped as _custom pages_. They cannot be customized in the UI Designer. Since they cannot be imported in a .bos archive, they do not come with the User Application you have downlaoded. Instead, they come as provided resources. You can find them in the "Resources" tab of the application.
```

Comment:
```suggestion
 * All pages previously created in other, more recent technologies, have been wrapped as _custom pages_. They cannot be customized in the UI Designer. Since they cannot be imported in a .bos archive, they do not come with the User Application you have downloaded. Instead, they come as provided resources. You can find them in the "Resources" tab of the application.
```